Windows/MainWindowMenu.cpp

```diff
@@ -328,30 +386,30 @@ namespace MainWindow {
 		}
 	}
 
-	void SaveStateActionFinished(bool result, void *userdata) {
+	static void SaveStateActionFinished(bool result, void *userdata) {
 		PostMessage(MainWindow::GetHWND(), WM_USER_SAVESTATE_FINISH, 0, 0);
 	}
 
-
+	// not static
```

Comment:
As you saw I moved the menu code out into its own file, but the main file still wants to call this function. Didn't want to rock the boat, cleanup later.